lke_node_pools.go

```diff
@@ -41,24 +60,30 @@ type LKENodePool struct {
 	Disks   []LKENodePoolDisk   `json:"disks"`
 	Linodes []LKENodePoolLinode `json:"nodes"`
 	Tags    []string            `json:"tags"`
+	Labels  LKENodePoolLabels   `json:"labels"`
+	Taints  []LKENodePoolTaint  `json:"taints"`
 
 	Autoscaler LKENodePoolAutoscaler `json:"autoscaler"`
 }
 
 // LKENodePoolCreateOptions fields are those accepted by CreateLKENodePool
 type LKENodePoolCreateOptions struct {
-	Count int               `json:"count"`
-	Type  string            `json:"type"`
-	Disks []LKENodePoolDisk `json:"disks"`
-	Tags  []string          `json:"tags"`
+	Count  int                `json:"count"`
+	Type   string             `json:"type"`
+	Disks  []LKENodePoolDisk  `json:"disks"`
+	Tags   []string           `json:"tags"`
+	Labels LKENodePoolLabels  `json:"labels"`
+	Taints []LKENodePoolTaint `json:"taints"`
 
 	Autoscaler *LKENodePoolAutoscaler `json:"autoscaler,omitempty"`
 }
 
 // LKENodePoolUpdateOptions fields are those accepted by UpdateLKENodePoolUpdate
 type LKENodePoolUpdateOptions struct {
-	Count int       `json:"count,omitempty"`
-	Tags  *[]string `json:"tags,omitempty"`
+	Count  int                 `json:"count,omitempty"`
+	Tags   *[]string           `json:"tags,omitempty"`
+	Labels *LKENodePoolLabels  `json:"labels,omitempty"`
+	Taints *[]LKENodePoolTaint `json:"taints,omitempty"`
```

Comment:
Do you think we make it a simple slice without pointer?
```suggestion
	Taints []LKENodePoolTaint `json:"taints,omitempty"`
```

Reply:
I think this needs to be a pointer because there is a case where a user wants to set their taints to an explicitly empty list

Reply:
@lgarber-akamai that's correct, we'll need it to be a pointer.